src/main/java/com/redhat/cloud/policies/app/model/history/PoliciesHistoryRepository.java

```diff
@@ -84,12 +84,14 @@ public List<PoliciesHistoryEntry> find(String tenantId, UUID policyId, Pager pag
 
```

Comment:
There are several method based on `tenantId` in this class that need to be duplicated to use the `orgId` instead. Calling one version or the other should be determined by the boolean from the configuration.

Reply:
done